client/dashboard/sites/settings-sftp-ssh/sftp-card.tsx

```diff
@@ -0,0 +1,175 @@
+import { DataForm } from '@automattic/dataviews';
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	BaseControl,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteSftpUsersResetPasswordMutation } from '../../app/queries';
+import ClipboardInputControl from './clipboard-input-control';
+import type { SftpUser } from '../../data/types';
+import type { Field } from '@automattic/dataviews';
+
+const SFTP_URL = 'sftp.wp.com';
+const SFTP_PORT = '22';
+
+type SftpCardFormData = {
+	url: string;
+	port: string;
+	username: string;
+	password: string;
+};
+
+export default function SftpCard( {
+	siteSlug,
+	sftpUsers = [],
+}: {
+	siteSlug: string;
+	sftpUsers: SftpUser[];
+} ) {
+	const { username = '', password = '' } = sftpUsers[ 0 ] ?? {};
+	const mutation = useMutation( siteSftpUsersResetPasswordMutation( siteSlug ) );
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+	const formData = {
+		url: SFTP_URL,
+		port: SFTP_PORT,
+		username,
+		password,
+	};
+
+	const handleCopy = ( label: string ) => {
+		createSuccessNotice(
+			sprintf(
+				/* translators: %s is the copied field */
+				__( 'Copied %s to clipboard.' ),
+				label
+			),
+			{
+				type: 'snackbar',
+			}
+		);
+	};
+
+	const ClipboardInputControlEdit = ( { field, data } ) => {
+		const { getValue } = field;
+		return (
+			<ClipboardInputControl
+				label={ field.label }
+				value={ getValue( { item: data } ) }
+				readOnly
+				__next40pxDefaultSize
+				onCopy={ handleCopy }
+			/>
+		);
+	};
+
+	const fields: Field< SftpCardFormData >[] = [
+		{
+			id: 'url',
+			label: __( 'URL' ),
+			Edit: ClipboardInputControlEdit,
+		},
+		{
+			id: 'port',
+			label: __( 'Port' ),
+			Edit: ClipboardInputControlEdit,
+		},
+		{
+			id: 'username',
+			label: __( 'Username' ),
+			Edit: ClipboardInputControlEdit,
+		},
+		{
+			id: 'password',
+			label: __( 'Password' ),
+			Edit: ( { field, data } ) => {
+				const { getValue } = field;
+				const value = getValue( { item: data } );
+				return value ? (
+					<ClipboardInputControl
+						label={ field.label }
+						value={ value }
+						help={ __(
+							'Save your password somewhere safe. You will need to reset it to view it again.'
+						) }
+						readOnly
+						__next40pxDefaultSize
+						onCopy={ handleCopy }
+					/>
+				) : (
+					<BaseControl
+						label={ __( 'Password' ) }
```

Comment:
This label should be all-caps like the rest of the labels, but I currently see it in sentence case in the default state.

<img width="583" alt="Screenshot 2025-06-03 at 11 47 25" src="https://github.com/user-attachments/assets/23bdb944-35b0-46e9-91d6-7d68e50ff9dc" />

It switches to all-caps when the password has been reset though.

<img width="580" alt="Screenshot 2025-06-03 at 12 10 23" src="https://github.com/user-attachments/assets/dbbbef20-3bf9-4d79-91e8-59ea44d9aa76" />